charts/prometheus-to-sd/templates/deployment.yaml

```diff
@@ -32,6 +32,9 @@ spec:
             {{- range $key, $value := .Values.metricsSources }}
             - --source={{ $key }}:{{ $value }}
             {{- end }}
+            {{- with .Values.monitoredResourceTypes }}
```

Comment:
Please add this into `values.yaml`. It could be commented out but include possible values and a short comment, if appropriate.

Reply:
fixed 4128e4e

I set this option to [default value](https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/prom-to-sd-v0.5.2/prometheus-to-sd/main.go#L55)